maxim/logger/llamaindex/utils.py

```
@@ -0,0 +1,103 @@
+            
+    @staticmethod
+    def parse_message_to_generation_result(
+        message: ChatMessage
+        ) -> GenerationResultMessage:
+        
+        parsed_messages = GenerationResultMessage(role=message.role, content=parse_message_content_to_generation_result(message.blocks), tool_calls=[])
+        
+        return parsed_messages
+
+    @staticmethod
+    def parse_model_parameters(llm: LLM) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)